src/app/contact/contact.component.scss

```diff
@@ -0,0 +1,70 @@
+.contact-us {
+  margin: 150px 0px;
+  padding: 30px 0px;
+}
+
+.header {
+  text-align: center;
+  margin: 30px 0px;
+}
+
+.header p {
+  font-size: 18px;
+}
```

Comment:
SCSS, w odróżnieniu od CSS, pozwala na nesting selektorów. Czyli zamiast
```css
.header {
    /* style */
}
.header p {
    /* style */
}
```
można zrobić
```css
.header {
    /* style */

    p {
        /* style */
    }
}
```

to się będzie powtarzało jeszcze wiele razy w tym projekcie, więc przerób wszystkie takie sytuacje.